editor/spatial_editor_gizmos.cpp

```diff
@@ -4082,6 +4082,21 @@ JointSpatialGizmoPlugin::JointSpatialGizmoPlugin() {
 	create_material("joint_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/joint", Color(0.5, 0.8, 1)));
 	create_material("joint_body_a_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/joint_body_a", Color(0.6, 0.8, 1)));
 	create_material("joint_body_b_material", EDITOR_DEF("editors/3d_gizmos/gizmo_colors/joint_body_b", Color(0.6, 0.9, 1)));
+
+	update_timer = memnew(Timer);
+	update_timer->set_name("JointGizmoUpdateTimer");
+	update_timer->set_wait_time(1.0/120.0);
```

Comment:
Style issue `(1.0 / 120.0)`

Reply:
Fixed, and I just set-up the git hook again... finally :P